docs/INSTALLATION.md

```diff
@@ -128,9 +128,11 @@ Onderstaande environment variabelen gaan over de bronnen die gekoppeld zijn aan
 | CONTACTMOMENTEN_BASE_URL  | URL van de Contactmomenten API voor het gebruikte <br />Contactmomentenregister  |
 | CONTACTMOMENTEN_API_CLIENT_ID | clientId voor het gebruikte Contactmomentenregister |
 | CONTACTMOMENTEN_API_KEY  | API key te gebruiken door KISS voor het gebruikte <br />Contactmomentenregister, deze moet **minimaal <br />16 karakters** lang zijn. |
-| ZAKEN_BASE_URL  |  URL waar de verschillende ZGW API's te benaderen zijn |
-| ZAKEN_API_CLIENT_ID  | clientId van de ZGW API's |
-| ZAKEN_API_KEY  | API Key van de ZGW API's, deze moet <br />**minimaal 16 karakters** lang zijn |
+| ZAAKSYSTEEM__0__BASE_URL  |  URL waar de verschillende ZGW API's te benaderen zijn. Er kunnen meerdere zaakystemen toegevoegd worden. |
+| ZAAKSYSTEEM__0__API_CLIENT_ID  | clientId van de ZGW API's |
+| ZAAKSYSTEEM__0__API_KEY  | API Key van de ZGW API's, deze moet <br />**minimaal 16 karakters** lang zijn |
+| ZAAKSYSTEEM__0__DEEPLINK_URL | Url van het zaaksysteem waarachter een kenmerk van een zaak geplakt kan worden om ernaar te deeplinken |

```

Comment:
zo te zien was deze property nog helemaal niet gedocumenteerd? volgens mij is deze optioneel. kunnen we dat erbij vermelden? 
als je een deeplink url hebt, dan is de deeplinkproperty wel verplicht neem ik aan. Dat lijkt me ook wel vermeldenswaardig